Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OuterTune/OuterTune strings](https://hosted.weblate.org/projects/outertune/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/outertune/app/horizontal-auto.svg)

vishussingh24 commented:
@lucasmz-dev 

Hi! Thank you for your guidance earlier. I’m still new to contributing to open source, and I’m learning step by step. Could you please let me know if I’m on the right path with my contributions so far?

Also, I wanted to ask—since I have a good understanding of Java, is it possible for me to fix bugs or make improvements directly in the codebase and create PRs for them? I would really appreciate any tips or suggestions you might have on how I can improve and contribute more effectively.

Thanks again for your support!

mikooomich commented:
> @lucasmz-dev
> 
> Hi! Thank you for your guidance earlier. I’m still new to contributing to open source, and I’m learning step by step. Could you please let me know if I’m on the right path with my contributions so far?
> 
> Also, I wanted to ask—since I have a good understanding of Java, is it possible for me to fix bugs or make improvements directly in the codebase and create PRs for them? I would really appreciate any tips or suggestions you might have on how I can improve and contribute more effectively.
> 
> Thanks again for your support!

Seems like your translation contribution went through see the commit list.

Yes, you can submit pull requests, the contributing guidelines are linked when you create a pull request, and also in CONTRIBUTING.MD. 

In terms of "what" improvments you can make, the issue tracker is a good list of things that can be worked on (if you have your own ideas, those are welcome too). Most issues labeled "bug" are available to be worked on, but not all issues that labeled "enhancement" are something we'd like to merge into the app (you can still submit pull requests for them if you'd like). If you have any specific questions, don't hesitate to ask!

vishussingh24 commented:
May i know  that, when this PR is going to merge?

lucasmz-dev commented:
Website PRs aren't usually merged right away, it is most likely to happen when a new official build/version is about to come out or something else where it makes sense.

weblate committed:
Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Translated using Weblate (Persian)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Hindi)

Currently translated at 66.3% (140 of 211 strings)

Translated using Weblate (Estonian)

Currently translated at 4.2% (9 of 211 strings)

Translated using Weblate (Arabic)

Currently translated at 71.5% (151 of 211 strings)

Translated using Weblate (English (Canada))

Currently translated at 0.9% (2 of 211 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Hebrew)

Currently translated at 37.4% (79 of 211 strings)

Translated using Weblate (Polish)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (German)

Currently translated at 33.1% (70 of 211 strings)

Translated using Weblate (Vietnamese)

Currently translated at 35.5% (75 of 211 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Greek)

Currently translated at 27.4% (58 of 211 strings)

Translated using Weblate (Bulgarian)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Azerbaijani)

Currently translated at 24.6% (52 of 211 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 59.7% (126 of 211 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Japanese)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (French)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (French)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (211 of 211 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (211 of 211 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Mickaël Binos <mickaelbinos@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ar/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/az/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/bg/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/de/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/el/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en_CA/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/es/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/et/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fa/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/he/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/hi/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/it/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ja/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ko/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pl/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/vi/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hans/
Translation: OuterTune/OuterTune strings

trunars committed:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/bg/

trunars committed:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/bg/

rekt0ro committed:
Translated using Weblate (Persian)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fa/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (212 of 212 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/
